recommendations/choices.py

```diff
@@ -226,217 +226,250 @@
+            "someone uses drugs",
+            "someone overdoses",
+        ],
+        "Drug Usage / Abuse",
+    ),
+    (
+        [
+            "Autism specific abuse",
+            "there's domestic violence",
+            "someone cheats",
+            "there's bisexual cheating",
+            "the abused forgives their abuser",
+            "someone gets gaslighted",
+            "someone is stalked",
+            "someone is abused with a belt",
+            "someone is sexually assaulted",
+            "someone is raped onscreen",
+            "someone is restrained",
+            "someone is drugged",
+            "there are incestuous relationships",
+        ],
+        "Abuse and Manipulation (including sexual)",
+    ),
+    (
+        [
+            "a woman gets slapped",
+            "someone is beaten up by a bully",
+            "teeth are damaged",
+            "someone struggles to breathe",
+            "someone breaks a bone",
+            "there's Achilles Tendon injury",
+            "hands are damaged",
+            "someone dislocates something",
+            "someone falls down stairs",
+            "somebody is choked",
+        ],
+        "Violence",
+    ),
+    (
+        [
+            "a woman is brutalized for spectacle",
+            "someone is held under water",
+            "there's torture",
+            "there's eye mutilation",
+            "there's excessive gore",
+            "there's genital trauma/mutilation",
+            "heads get squashed",
+            "there's finger/toe mutilation",
+            "there's body horror",
+            "there's cannibalism",
+            "someone is eaten",
+            "someone is burned alive",
+            "there are hangings",
+            "there's decapitation",
+            "someone is crushed to death",
+            "there's amputation",
+            "someone is buried alive",
+            "someone's throat is mutilated",
+            "someone falls to their death",
+            "someone is stabbed",
+            "someone dies by suicide",
+            "someone drowns",
+            "a car crashes",
+            "a plane crashes",
+            "there's blood/gore",
+            "there's gun violence",
+            "a person is hit by a car",
+            "someone asphyxiates",
+            "someone is kidnapped",
+            "Someone attempts suicide",
+            "someone self harms",
+            "There's audio gore",
+        ],
+        "Excessive Violence",
+    ),
+    (
+        [
+            "someone dies",
+            "someone sacrifices themselves",
+            "a family member dies",
+            "a parent dies",
+        ],
+        "Human Death",
+    ),
+    (
+        [
+            "someone poops on-screen",
+            "someone vomits",
+            "someone wets/soils themselves",
+            "there's spitting",
+            "there's farting",
+            "there's menstruation",
+        ],
+        "Human Waste",
```

Comment:
I think this should be changed to Bodily Functions as it better represents this collection of triggers